specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/azureFirewall.json

```diff
@@ -473,6 +473,128 @@
          },
          "description":"Properties of an application rule."
       },
+      "AzureFirewallNatRuleCollectionPropertiesFormat": {
+         "properties":{  
+            "priority":{  
+               "type":"integer",
+               "format":"int32",
+               "maximum":65000,
+               "exclusiveMaximum":false,
+               "minimum":100,
+               "exclusiveMinimum":false,
+               "description":"Priority of the NAT rule collection resource."
+            },
+            "action":{  
+               "$ref":"#/definitions/AzureFirewallNatRCAction",
+               "description":"The action type of a NAT rule collection"
+            },
+            "rules":{  
+               "type":"array",
```

Comment:
Please check if any of these properties are readonly and mark them accordingly

Reply:
I marked etag. Any of the rest can be updated by user.